service/assessment/assessment_test.go

```diff
@@ -322,14 +322,19 @@ func TestAssessmentResultHooks(t *testing.T) {
 		hookCallCounter = 0
 	)
 
+	var wg sync.WaitGroup
+	wg.Add(12)
+
 	firstHookFunction := func(assessmentResult *assessment.AssessmentResult, err error) {
 		hookCallCounter++
 		log.Println("Hello from inside the firstHookFunction")
```

Comment:
I know it was there already, but remove logging/printing stuff in tests?

Reply:
Why?